### PR TITLE
Update get-ti-indicators-collection.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated odata filter queries

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5889

File: microsoft-365/security/defender-endpoint/get-ti-indicators-collection.md [hunk 1]
```diff
@@ -36,7 +36,9 @@ Retrieves a collection of all active [Indicators](ti-indicator.md).
 
 Supports [OData V4 queries](https://www.odata.org/documentation/).
 
-The OData's `$filter` query is supported on: `indicatorValue`, `indicatorType`, `creationTimeDateTimeUtc`, `createdBy`, `action` and `severity` properties.
+The OData's `$filter` query is supported on: `application`, `createdByDisplayName`, `expirationTime`, `generateAlert`, `title`, `rbacGroupNames`, `rbacGroupIds`, `indicatorValue`, `indicatorType`, `creationTimeDateTimeUtc`, `createdBy`, `action` and `severity` properties.
+<br>```$top``` with max value of 10,000 
+<br>```$skip```
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/get-ti-indicators-collection.md]:
```suggestion
The OData's `$filter` query is supported on: `application`, `createdByDisplayName`, `expirationTime`, `generateAlert`, `title`, `rbacGroupNames`, `rbacGroupIds`, `indicatorValue`, `indicatorType`, `creationTimeDateTimeUtc`, `createdBy`, `action`, and `severity` properties.
<br>```$top``` with max value of 10,000. 
<br>```$skip```.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @msbemba - kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**